libs/dh/market-participant/feature-organization/src/lib/market-roles-overview/dh-market-participant-market-roles-overview.component.html

```diff
@@ -0,0 +1,44 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<watt-card *transloco="let t; read: 'marketParticipant.marketRolesOverview'">
+  <watt-card-title>
+    <div class="card-title-container">
+      <h4>{{ count + " " + t("marketRoles") }}</h4>
```

Comment:
suggestion: instead of having a separate variable, we can use `dataSource` directly.

```suggestion
      <h4>{{ dataSource.data.length + " " + t("marketRoles") }}</h4>
```

This will also get rid of this error:

![image](https://user-images.githubusercontent.com/1096332/217261138-043c8b77-fc0e-4da5-bf6c-a1f640b2792e.png)